webrtc.html

```diff
@@ -1840,8 +1840,23 @@
                 system. The session descriptions MUST remain usable by
                 <code>setLocalDescription</code> without causing an error until
                 at least the end of the fulfillment callback of the returned
-                promise. Calling this method is needed to generate the ICE
-                username fragment and password.</p>
+                promise.</p>
+                <p>As an answer, the generated SDP will contain a specific
+                codec/RTP/RTCP configuration that, along with the corresponding offer,
+                specifies how the media plane should be established. The
+                generation of the SDP MUST follow the appropriate process for
+                generating an answer described in [[!JSEP]].</p>
+                <p>The generated SDP will also contain the <a>ICE agent</a>'s
+                <a data-link-for="RTCIceParameters">usernameFragment</a>, 
+                <a data-link-for="RTCIceParameters">password</a> and
+                ICE options (as defined in [[!ICE]], Section 14) and may also contain
+                any local candidates that have been gathered by the agent.</p>
+                <p>The value for <code>certificates</code> in the
+                <code><a>RTCConfiguration</a></code> for the
+                <code>RTCPeerConnection</code> is used to produce a set of
+                certificate fingerprints. These certificate fingerprints are
+                used in the construction of SDP and as input to requests for
+                identity assertions.</p>
```

Comment:
This isn't quite correct, since automatically generated certificates don't appear in the RTCConfiguration. But this isn't a new issue since this is just copying text from `createOffer`.